test/test_evaluation/test_train_evaluator.py

```diff
@@ -557,12 +557,16 @@ def test_file_output(self, loss_mock):
         configuration = unittest.mock.Mock(spec=Configuration)
         queue_ = multiprocessing.Queue()
         loss_mock.return_value = None
+        scorer_list = ['accuracy', 'balanced_accuracy', 'f1_macro', 'f1_micro',
```

Comment:
Could you please extract this so that it can be reused by multiple tests?

Reply:
Refactored code.